src/gui/qgsattributeform.cpp

```diff
@@ -1346,8 +1348,8 @@ void QgsAttributeForm::synchronizeState()
 
   for ( QgsWidgetWrapper *ww : std::as_const( mWidgets ) )
   {
-    QgsEditorWidgetWrapper *eww = qobject_cast<QgsEditorWidgetWrapper *>( ww );
-    if ( eww )
+
+    if ( QgsEditorWidgetWrapper *eww = qobject_cast<QgsEditorWidgetWrapper *>( ww ); eww )
```

Comment:
why this change? I find this less readable.

Reply:
It doesn't need the "; eww" part, it works without that